src/features/island/hud/components/deposit/lib/depositPreference.ts

```diff
@@ -0,0 +1,9 @@
+const host = window.location.host.replace(/^www\./, "");
+const DEPOSIT_PREFERENCE_KEY = `depositPreference.${host}-${window.location.pathname}`;
+export const getDepositPreference = () => {
+  return Number(localStorage.getItem(DEPOSIT_PREFERENCE_KEY));
+};
+
+export const setDepositPreference = (chainId: number) => {
+  localStorage.setItem(DEPOSIT_PREFERENCE_KEY, chainId.toString());
+};
```

Comment:
This is used for players depositing from an external wallet, so the app wont keep switching back chains each time they enter the deposit screen.